pkg/nl/list.go

```diff
@@ -8,6 +8,48 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func (n *NetlinkManager) listRoutes() ([]netlink.Route, error) {
+	routes, err := netlink.RouteListFiltered(netlink.FAMILY_ALL, &netlink.Route{
+		Table: 0,
+	}, netlink.RT_FILTER_TABLE)
+	if err != nil {
+		return nil, err
+	}
+	return routes, nil
+}
+
+func (n *NetlinkManager) listNeighbors() ([]netlink.Neigh, error) {
+	neighbors, err := netlink.NeighList(0, netlink.FAMILY_ALL)
+	if err != nil {
+		return nil, err
```

Comment:
Wrap error

Reply:
could you elaborate how I can fix the wrap error?

Reply:
Please see the comment here: https://github.com/telekom/das-schiff-network-operator/pull/4#discussion_r1312988511